lua/core/settings.lua

```diff
@@ -114,4 +114,8 @@ settings["dap_deps"] = {
 	"python", -- Python (debugpy)
 }
 
+local ok, user_settings = pcall(require, "user.settings")
+if ok then
+	settings = vim.tbl_deep_extend("force", settings, user_settings)
+end
```

Comment:
Since this pattern is used in many places, why not implement this as a function? E.g.:
```suggestion
require("...").config_extend(settings, "user.settings")
```

Reply:
Created a new `config_extend` function in `modules.utils`.
This resulted in a unified description.